README.md

```diff
@@ -144,101 +141,72 @@ See [@typescript-eslint/parser's README.md](https://github.com/typescript-eslint
+| [no-return-void](docs/rules/no-return-void.md)                     | Disallow functions that don't return anything. | ![badge-lite][] ![badge-no-statements][] ✅ |     |                 |     |
 
-| Name                                                             | Description                               | <span title="Currying">:see_no_evil:</span> | <span title="Lite">:hear_no_evil:</span> | <span title="Recommended">:speak_no_evil:</span> | :wrench: | :blue_heart: |
-| ---------------------------------------------------------------- | ----------------------------------------- | :-----------------------------------------: | :--------------------------------------: | :----------------------------------------------: | :------: | :----------: |
-| [`functional-parameters`](./docs/rules/functional-parameters.md) | Functions must have functional parameters |             :heavy_check_mark:              |            :heavy_check_mark:            |                :heavy_check_mark:                |          |              |
+### Stylistic
 
-### Stylistic Rules
+| Name                                       | Description                         | 💼                   | ⚠️  | 🚫                 | 🔧  |
+| :----------------------------------------- | :---------------------------------- | :------------------- | :-- | :----------------- | :-- |
+| [prefer-tacit](docs/rules/prefer-tacit.md) | Replaces `x => f(x)` with just `f`. | ![badge-stylistic][] |     | ![badge-lite][] ✅ | 🔧  |
 
-:see_no_evil: = `stylistic` Ruleset.
+<!-- end auto-generated rules list -->
 
-| Name                                           | Description             | <span title="Stylistic">:see_no_evil:</span> | <span title="Lite">:hear_no_evil:</span> | <span title="Recommended">:speak_no_evil:</span> | :wrench: | :blue_heart: |
-| ---------------------------------------------- | ----------------------- | :------------------------------------------: | :--------------------------------------: | :----------------------------------------------: | :------: | :----------: |
-| [`prefer-tacit`](./docs/rules/prefer-tacit.md) | Tacit/Point-Free style. |              :heavy_check_mark:              |            :heavy_check_mark:            |                :heavy_check_mark:                | :wrench: | :blue_heart: |
+[badge-currying]: https://img.shields.io/badge/-currying-red.svg
+[badge-lite]: https://img.shields.io/badge/-lite-green.svg
+[badge-no-exceptions]: https://img.shields.io/badge/-no--exceptions-blue.svg
+[badge-no-mutations]: https://img.shields.io/badge/-no--mutations-orange.svg
+[badge-no-object-orientation]: https://img.shields.io/badge/-no--object--orientation-yellow.svg
+[badge-no-statements]: https://img.shields.io/badge/-no--statements-purple.svg
+[badge-off]: https://img.shields.io/badge/-off-black.svg
+[badge-stylistic]: https://img.shields.io/badge/-stylistic-gold.svg
```

Comment:
We use badges to represent most configs. Alternatively, the tools allow an emoji to be specified for each config as desired.